cmd/pfcpsim-client/main.go

```diff
@@ -26,20 +26,22 @@ type PFCPClientContext struct {
 	pdrs []*ieLib.IE
 	fars []*ieLib.IE
 	qers []*ieLib.IE
+
+	downlinkTEID uint32 // needed when updating FAR
 }
 
 var (
 	remotePeerAddress net.IP
 	upfAddress        net.IP
 	nodeBAddress      net.IP
-	ueAddressPool     string
+	ueAddressPool     *string
 
 	lastUEAddress net.IP
 
-	inputFile  string
-	outputFile string
+	inputFile  *string
+	outputFile *string
 
-	sessionCount int
+	sessionCount *int
```

Comment:
Switched to pointers, otherwise will get always default values from flag parsing